docs/command-line.md

```diff
@@ -370,6 +372,12 @@ See [The LibIdentify repo for more information and examples](https://github.com/
 Will cause the executable to print a message and wait until the return/ enter key is pressed before continuing -
 either before running any tests, after running all tests - or both, depending on the argument.
 
+<a id="skip-benchmarks"></a>
+## Skip all benchmarks
+<pre>--skip-benchmarks</pre>
+
+When this option is set, all benchmarks of each and every Catch test case will be skipped so that it appears as if there were no benchmark defined.
```

Comment:
How about

```suggestion
This flag tells Catch2 to skip running all benchmarks. Benchmarks in this
case mean code blocks in `BENCHMARK` and `BENCHMARK_ADVANCED` macros, not
test cases with the `[!benchmark]` tag.
```

I think this is clearer. (Also new docs use Catch2 and line length of 80, to play nicer with diffs)